app/assets/stylesheets/govuk-component/_govspeak-html-publication.scss

```diff
@@ -14,83 +12,23 @@
     text-align: start;
   }
 
-  .govuk-govspeak {
-    @include media(tablet) {
-      width: 75%;
-      float: right;
-
-      &.direction-rtl {
-        float: left;
-      }
+  @include media(false, $desktop-breakpoint) {
+    .govuk-sticky-element--enabled {
+      position: static;
     }
 
-    .number {
-      background: $white;
+    .offset-one-quarter {
```

Comment:
This `offset-one-quarter` looks like a reset of some styles, but where is the original thing it's resetting?